### PR TITLE
docs: improve formatting for CONTRIBUTING

### PR DESCRIPTION
**Pull request by acrisci**

### CODE REVIEW

**Comment by acrisci** [comment]
https://github.com/acrisci/i3/blob/feature/docs-contributing-formatting/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+## i3status/i3lock bug reports and feature requests
 
-# i3 bugreports/feature requests
+Note that [i3status](https://github.com/i3/i3status) and [i3lock](https://github.com/i3/i3lock) related bugreports and feature requests should be
+filed in the corresponding repositories.
 
-1. Read http://i3wm.org/docs/debugging.html
+## i3 bug reports and feature requests
+
+1. Read the [debugging docs](http://i3wm.org/docs/debugging.html).
```
Comment (stapelberg) [inline CONTRIBUTING.md]: “docs” makes it sound like there might be a lot of documentation, whereas the instructions are intentionally rather brief. Maybe replace by “instructions”?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,16 +16,16 @@ and https://github.com/i3/i3lock
    experience has shown that often, the software in question is responsible for
    the issue. Please raise an issue with the software in question, not i3.
 
-# Pull requests
+## Pull requests
 
 * Before sending a pull request for new features, please check with us that the
   feature is something we want to see in i3 by opening an issue which has
-  “feature request” or “enhancement” in its title.
+  *feature request* or *enhancement* in its title.
```
Comment (stapelberg) [inline CONTRIBUTING.md]: Why this change? I like the quotation marks better, since they communicate that the word has to appear verbatim.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+# Contributing
 
-Note that i3status and i3lock related bugreports and feature requests should be
-filed in the corresponding repositories, i.e. https://github.com/i3/i3status
-and https://github.com/i3/i3lock
+## i3status/i3lock bug reports and feature requests
 
-# i3 bugreports/feature requests
+Note that [i3status](https://github.com/i3/i3status) and [i3lock](https://github.com/i3/i3lock) related bugreports and feature requests should be
+filed in the corresponding repositories.
```
Comment (stapelberg) [inline CONTRIBUTING.md]: On the fence about this change. It might not be entirely obvious that the linked names lead to the github repositories.

**Comment by acrisci** [comment]
Ok see if this is better.
